packages/manager/modules/octavia-load-balancer/src/load-balancers/load-balancer/listeners/components/listener-form/template.html

```diff
@@ -59,7 +59,7 @@
             data-model="$ctrl.model.pool"
             data-match="name"
             data-items="$ctrl.filteredPools"
-            data-disabled="$ctrl.model.protocol === $ctrl.PROTOCOLS.PROMETHEUS"
+            data-disabled="$ctrl.filteredPools.length === 0"
```

Comment:
not sure it's working when `filteredPools` is null (to be confirmed).
Anyway, that one works on both
```suggestion
            data-disabled="!$ctrl.filteredPools.length"
```